src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFHeaderLines.java

```diff
@@ -77,7 +77,7 @@ public static VCFFormatHeaderLine getEquivalentFormatHeaderLine(final String inf
         addFilterLine(new VCFFilterHeaderLine(VCFConstants.PASSES_FILTERS_v4, "Site contains at least one allele that passes filters"));
 
         addFilterLine(new VCFFilterHeaderLine(NAY_FROM_YNG, "Considered a NAY in the Yay, Nay, Grey table"));
-        addFilterLine(new VCFFilterHeaderLine(EXCESS_ALLELES, "Site has an excess (>" + CommonCode.EXCESS_ALLELES_THRESHOLD + ") of alternate alleles"));
+        addFilterLine(new VCFFilterHeaderLine(EXCESS_ALLELES, "Site has an excess of alternate alleles based on the input threshold (default is 6)"));
```

Comment:
I'd keep the  reference to common code value like `"Site has an excess of alternate alleles based on the input threshold (default is " + CommonCode.EXCESS_ALLELES_THRESHOLD + ")"`

Reply:
I really liked how informative your VQSR header lines are @mmorgantaylor it would be great if we can use the actual threshold value here.  Take a look at how this is done in ExtractCohort, though you can just do it directly there (you don't need an equivalent of FilterSensitivityTools)